### ISSUE TITLE
Accepting a suggestion causes VSCode to "forget" about auto-close pairs

### ISSUE THREAD

**Issue by thisisrandy** [codota/tabnine-vscode#60]
By way of example (python). Assume the current text is
```
foo = "foo"
print|
```
where `|` is the cursor position. Type `(f`
```
foo = "foo"
print(f|)
```
Accept the suggestion `foo`
```
foo = "foo"
print(foo|)
```
Type `)`
```
foo = "foo"
print(foo)|)
```
Likewise for double quotes or any other auto-close pair character. Type `"f`
```
"f|"
```
Accept the suggestion `foo`
```
"foo|"
```
Type `"`
```
"foo"|"
```
The expected behavior, if it isn't clear, is for the out quote/paren/etc. to be recognized as having been automatically inserted and written over when typed explicitly. This works as expected with extensions disabled or when no suggestion is accepted, so it appears to be a problem with the tabnine extension specifically.

Please let me know if I can provide any additional information.

**Comment by scascketta** [comment]
I think this is the same issue as described in #47. I am irritated by this as well.

**Comment by thisisrandy** [comment]
Good catch. I missed that one. I'll leave it up to @zxqfl to decide which issue he wants to address this on.

**Comment by dimacodota** [comment]
the fix was released in version 2.2.3
